File: JavaScript/프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    for (const command of commands){
        const [i,j,k] = command;
        let tempArr = array.slice(i - 1, j);
        tempArr.sort((a,b) => { return a - b});
        answer.push(tempArr[(k - 1) % tempArr.length])
    }
    return answer;
}

File: C++/프로그래머스/1/42748. K번째수/K번째수.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    for (auto command: commands){
        vector<int> ref(array.begin() + command[0] - 1, array.begin() + command[1]);
        sort(ref.begin(), ref.end());
        answer.push_back(ref[command[2] - 1]);
    }
    return answer;
}